src/utils/__snapshots__/PilotPromptCreator.test.ts.snap

```diff
@@ -0,0 +1,264 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PilotPromptCreator should create a prompt for an intent correctly 1`] = `
+"# Next Step Generation
+
+You are an AI assistant tasked with predicting the next optimal action a user should take within an application to progress towards a specific goal.
+Please generate a one-line string that precisely describes the next action the user should take to move closer to their goal, and another string (which can be greater than one line) which describes your toughts while creating the step
+If you think that the goal has been reached return a one word 'success'
+
+## Context
+
+### Intent to perform
+
+Generate a one-line string that precisely describes the next action the user should take to move closer to their goal which is: "tap button"
```

Comment:
not sure if this is needed cause we already have it under the `## Instructions` and the general overview